ci/kokoro/gcloud-functions.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+if [[ -z "${GCLOUD_CONFIG:-}" ]]; then
+  readonly GCLOUD_CONFIG="cloud-cpp-integration"
+  readonly GCLOUD_ARGS=(
+      # Do not seek confirmation for any actions, assume the default
+      "--quiet"
```

Comment:
Should the `GCLOUD_ARGS=( '--quiet' )` be outside the `${GCLOUD_CONFIG}` guard?

Even if not, should `gcloud-functions.sh` always be setting some value for `${GCLOUD_ARGS}`?

Reply:
Fixed, I think, but let me know if I missed the point.

Reply:
I was thinking you might just want something along the lines of ...
```
if [[ -z "${GCLOUD_CONFIG:-}" ]]; then
  GCLOUD_CONFIG="cloud-cpp-integration"
fi
readonly GCLOUD_CONFIG

GCLOUD_ARGS+=( "--quiet" "--configuration=${GCLOUD_CONFIG}" )
readonly GCLOUD_ARGS
```

Reply:
I think the `GCLOUD_ARGS+=` would fail if you source the file twice?

Reply:
I see ... the `-z` tests are not to allow caller overrides, but rather to allow repeated `source`ing.  OK.  Thanks.